main.h

```diff
@@ -30,34 +23,43 @@ struct UserData { // Struct used for storage of the users info
 
 };
 
-class ItemsProperties{
+class Items{
     public:
     std::string name;
     int call_code;
+
+    Items(const std::string received_name, int received_call_code) {
+      name = received_name;
+      call_code = received_call_code;
+    }
```

Comment:
nit: insert linebreak

Reply:
Sorry, didn't understood this one

Reply:
Just a linebreak between function definitions.
```suggestion
    }

```